avidemux_tr.ts

```diff
@@ -6322,60 +6963,123 @@ Yüklemek istiyor musunuz  ?
         <translation>TS Çoklayıcı</translation>
     </message>
 </context>
+<context>
+    <name>ffv1</name>
+    <message>
+        <location filename="../../../avidemux_plugins/ADM_videoEncoder/ffv1/ADM_ffv1Encoder.cpp" line="126"/>
+        <source>Golomb-Rice</source>
+        <translation>Golomb-Pirinç</translation>
```

Comment:
Thank you, but Robert Rice, the author of this method, is not groats. Please fix.

Reply:
Thank you for the warning. I fixed it as Golomb-Rice